Packs/CortexXDR/Integrations/CortexXDRIR/README.md

```diff
@@ -7,33 +7,38 @@ This integration was integrated and tested with version 2.6.5 of Cortex XDR - IR
+    | **Parameter** | **Description** | **Required** |
+    | --- | --- | --- |
+    | Fetch incidents |  | False |
+    | Incident type |  | False |
+    | Server URL (copy URL from XDR) |  | True |
+    | API Key ID |  | False |
+    | API Key |  | False |
+    | Remove legacy incident fields | Unchecked for backwards compatibility, recommended to check. This will remove duplicated incident fields under file_artifacts, network_artifacts, and alerts (like client_id, clientid.) | False |
+    | Incident Mirroring Direction |  | False |
+    | Close Mirrored XSOAR Incident | When selected, closing the Cortex XDR incident is mirrored in Cortex XSOAR. | False |
+    | Close Mirrored Cortex XDR Incident | When selected, closing the Cortex XSOAR incident is mirrored in Cortex XDR. | False |
+    | XDR mirroring delay in minutes | In the event of a delay in mirroring incoming changes from XDR, use the xdr_delay parameter to extend the look-back period. However, be aware that this may result in increased latency when updating incidents. | False |
+    | Custom close-reason mapping (XSOAR -&gt; XDR mirrored incident. Overwrites default close-reason mapping defined by Cortex XSOAR) | Define how to close the mirrored incidents from Cortex XSOAR into Cortex XDR with a custom close reason mapping. Enter a comma-separated list of close reasons \(acceptable format \{Cortex XSOAR close reason\}=\{Cortex XDR close reason\}\) to override the default close reason mapping defined by Cortex XSOAR. Note that the mapping must be configured accordingly with the existing close reasons in Cortex XSOAR and Cortex XDR. Not following this format will result in closing the incident with a default close reason. Example: "Resolved=Other,Duplicate=Other". Refer to ../README.md for possible close-reasons - \`XDR Incident Mirroring\`. | False |
+    | Custom close-reason mapping (XDR -&gt; XSOAR mirrored incident. Overwrites default close-reason mapping defined by Cortex XSOAR) | Define how to close the mirrored incidents from Cortex XDR into Cortex XSOAR with a custom close reason mapping. Enter a comma-separated list of close reasons \(acceptable format \{Cortex XDR close reason\}=\{Cortex XSOAR close reason\}\) to override the default close reason mapping defined by Cortex XSOAR. Note that the mapping must be configured accordingly with the existing close reasons in Cortex XSOAR and Cortex XDR. Not following this format will result in closing the incident with a default close reason. Example: “Known Issue=Resolved, Duplicate Incident=Other". Refer to ../README.md for possible close-reasons - \`XDR Incident Mirroring\`. | False |
+    | API Key ID |  | False |
+    | API Key |  | False |
```

Comment:
It seems like it exists twice. 
Please check that it looks ok - https://github.com/demisto/content/blob/3446769845fa66ae5c7964674901a0293acf179c/Packs/CortexXDR/Integrations/CortexXDRIR/README.md
Verify the only change is the added params

Reply:
Deleted the duplicated fields and compared the old README with the new one.